### DISCUSSION TITLE
Unecessary lint check and install golangci-lint jobs for build-linux pipeline

### DISCUSSION THREAD

**Discussion by amirvalhalla** [pactus-project/pactus#694] [General]
Hi
based on we have separate pipeline for linting, I think we don't need `lint check` and `install golangci-lint` for `build-linux` pipeline.

what is your opinion?

@b00f 
@themantre

**Comment by b00f** [comment]
Where exactly? Which pipeline?

**Reply by amirvalhalla** [reply]
@b00f 

see [this](https://github.com/pactus-project/pactus/actions/runs/6164014505/job/16728816721?pr=692) pipeline

**Reply by b00f** [reply]
It checks linting for GUI project

**Reply by amirvalhalla** [reply]
our `lint check` pipeline won't check for GUI?

**Reply by b00f** [reply]
No, Because GUI is tagged . GUI is too slow to include in our main build
